liberty-ls/src/main/resources/DiagnosticMessages.properties

```diff
@@ -2,4 +2,5 @@ INVALID_PROPERTY_VALUE=The value `{0}` is not valid for the property `{1}`.
 INVALID_VARIABLE_VALUE=The value `{0}` is not valid for the variable `{1}`.
 INVALID_PROPERTY_INTEGER_RANGE=The value `{0}` is not within the valid range `{2}` for the property `{1}`.
 INVALID_VARIABLE_INTEGER_RANGE=The value `{0}` is not within the valid range `{2}` for the variable `{1}`.
-INVALID_PACKAGE_LIST_FORMAT=This value must be a comma-delimited list of Java packages.
\ No newline at end of file
+INVALID_PACKAGE_LIST_FORMAT=This value must be a comma-delimited list of Java packages.
+SERVER_ENV_INVALID_WHITESPACE=There should be no whitespace surrounding the equals sign (=).
```

Comment:
I see you copied this from the IBM doc, but you added an 's' to 'equal'.  Probably should just say 'equal'.